aten/src/ATen/core/Formatting.cpp

```diff
@@ -238,7 +238,17 @@ std::ostream& print(std::ostream& stream, const Tensor & tensor_, int64_t linesi
     stream << "size:\n" << tensor_.sizes() << "\n";
     stream << "]";
   } else {
-    Tensor tensor = tensor_.to(kCPU, kDouble).contiguous();
+    Tensor tensor;
+    if (tensor_.is_quantized()) {
```

Comment:
let's discuss here, so what about toString?

Reply:
toString only prints the type of the tensor. So in this case it prints `QuantizedCPUQUInt8Type{}`

Reply:
oh looks like it only gives the type of Tensor?